### ISSUE TITLE
nether

### ISSUE THREAD

**Issue by Rocarice** [justinwon777/HumanCompanions#13]
Hey, I really like the mod! But I have a problem. 
I uses 3 of these guy to protect my base. 2 Archer, one swordman, all on "guard" I dont take them on journeys, cause I lost already 3 of them to falldamage. 
But when I travel to the nether, they're following me trough the portal, no matter what AND - they run into one direction and I have to catch em, to set them on follow again. I tried to take them a bit more far away from that portalarea, but they're still following. 
They shouldnt be in the nether, they have a snowballschance in hell there :D

**Comment by justinwon777** [comment]
That's strange. Tried to reproduce it myself but I couldn't. Can you post pictures on how your nether portal's setup and where the companions are?

**Comment by Rocarice** [comment]
Yes! I've made a copy of the savegame before I stepped through the portal again. I'm uploading a video where you can see what happens. I explain it in the video.
The mod is in my own private modpack, no datapacks used.
oh and one question. If one of there guys would die, would they respawn (or another new companion) at their house or are they gone forever? It would be great, because I have to travel far to find them and it's hard to bring them home, if there are mountains near by, cause of the falldamage.
the videolink https://youtu.be/gGpOjgNZM9g

**Comment by justinwon777** [comment]
Thanks for the video. Weird that they teleport with you to the nether and back out. For me they have to walk through the portal. If you set them to sitting (shift right click) Do they still follow? Also do you have other pets? Wondering if other tamed animals also follow you to the nether.

**Comment by Rocarice** [comment]
Hey Justin, thanks for the answer. I will try out the option with the sittingmode. No, the other tamed animal dont follow into the nether. There are 5 cats and 2 Wolves at the moment. I remember a very old bug from the vanilla MC, about 10 years ago, when your pets teleport to your side when you were in battle. But the companions stay when I'm in a fight somewhere else.
But yesterday I took a short trip to the woods when I was looking for Bees. I uses the Beenestlocator from productive bees. I dont know if it had something to do with it but when I right clicked the ground with it, suddenly all the guys appeared on my side and startet to run in the direction back home *lol* A fourth guy that I've found yesterday, is still missing.
I've also made another savegame for testing bees. There I will try to get some companions too, so I will see, if it also happens in the other savegame. I will let you know!
